### DISCUSSION TITLE
Third-party Theme Support Question

### DISCUSSION THREAD

**Discussion by andreaxhearts** [Maplespe/DWMBlurGlass#394] [Q&A]
The readme does state third-party theme support, however I can't get DWMBG to work with a specific custom theme. File explorer has a solid background, no transparency. Active/inactive text color can be changed however.

I'm using [dpcdpc11's](https://www.deviantart.com/dpcdpc11) themes, specifically _Simplify 10 Vanilla_. I don't know much about how themes work or how they are put together. But if anyone knows if there's any changes I can make to get it to work, it would be welcome! Otherwise, no big deal; just wondering.

The Windows Aero theme displays the glass effects fine, so DWMBlurGlass is installed correctly.

One thing to note is that when you hover the cursor over a _taskbar thumbnail_ for an explorer window, the preview of the window shows the glass effects! Also that hides the caption buttons, so could be related.

 > OS Version : Windows 10 22H2.19045.4894
 > Software Version : 2.2.0
 > Symbol files: Valid
 > Default DWMBG settings
 > Either Light mode or Dark mode.

**What other third-party software do you use:**
Tested separately with this configuration,
 > Windows 10 22H2.19045.2006
 > UltraUXTheme Patcher
 >
 > dpcdpc11 themes:
 > Simplify_10_Vanilla_Light_Squared.theme = no effects
 > Simplify_10_Sunset_Light.theme = no effects
 > Maverick_11_Light.theme = glass effects

**Comment by andreaxhearts** [comment]
![explorer screenshot with taskbar thumbnail preview](https://github.com/user-attachments/assets/007687d3-3231-44c8-a1a7-56ba30da0e91)

**Comment by ALTaleX531** [comment]
The theme renders solid colors to cover the blur, you need to edit specific part of your theme with msstyleEditor.

**Comment by Undisputed00x** [comment]
The best ready to go theme which works nicely with background effects is [Rectify11](https://github.com/Rectify11/Installer/tree/legacy-installer/Resources/Themes/themes)
